### PR TITLE
Flowser explorer integration

### PR DESCRIPTION
**Pull request by sideninja**
Closes: #501 #698 

## Description
This PR integrates the Flowser tool: https://github.com/onflowser/flowser
https://flowser.dev/

It handles the installation, configuration, and running of the tools. It also provides a way to run in the context of an existing project.

Depending on: https://github.com/onflowser/flowser/pull/139

______

For contributor use:

- [ ] Targeted PR against `master` branch
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work
- [ ] Code follows the [standards mentioned here](https://github.com/onflow/flow-cli/blob/master/CONTRIBUTING.md#styleguides)
- [ ] Updated relevant documentation
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Added appropriate labels

### CODE REVIEW

File: internal/tools/flowser.go [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+		}
+	}
+
+	projectPath, err := os.Getwd()
+	if err != nil {
+		return nil, err
+	}
+
+	// check if current directory is existing flow project if not then don't pass project path to Flowser, so user can choose a project
+	_, err = state.ReadFile("foo")
```
Comment (bthaile) [inline internal/tools/flowser.go:85]: "foo" just looks weird, "dummyPath" or something more descriptive would be more intuitive.

Comment (sideninja) [reply internal/tools/flowser.go:85]: ohh sorry, that's not supposed to be there... still doing some testing, my bad. Thanks for catching

**Review by bthaile — APPROVED** [review]
lgtm, one comment for clarification on directory check.

**Review by DylanTinianov — APPROVED** [review]
Looks good

**Pull request merged**